documents/scitools_licensing.md

```diff
@@ -21,7 +21,7 @@ Copyright <start year>(-<end year>) <project> contributors
 For example:
 
 ```
-Copyright 2010-2018 Iris contributors
+Copyright 2010-2019 Iris contributors
```

Comment:
Not strictly necessary but it got picked up when I was changing the others, and I kept as we might as well also update this.